### DISCUSSION TITLE
How to change the label text color?

### DISCUSSION THREAD

**Discussion by fredrikburmester** [nandorojo/zeego#112] [General]
<img width="180" alt="Screenshot 2024-11-10 at 17 20 02" src="https://github.com/user-attachments/assets/e0b8fab5-edf9-44a3-bc25-740a60f7ac9a">

**Comment by nandorojo** [comment]
Is this android?

**Reply by fredrikburmester** [reply]
Yes

**Reply by fredrikburmester** [reply]
And i have no clue how to change the text color.

**Reply by nandorojo** [reply]
It’s up to android so something must be wrong there because it’s a native component

**Reply by nandorojo** [reply]
Maybe some color scheme issue in your app config? Not sure honestly. Maybe a react-native-menu problem

**Reply by fredrikburmester** [reply]
Is it not possible to overwrite the color of the text? That would really help.

**Reply by wcastand** [reply]
i have the same issue currently and can't figure it out, @fredrikburmester did you found a solution or the root of the issue by any chance?
